### ISSUE TITLE
[Supporting the platform  - Query Pattern]: Controls overlap with Grafana cloud-ml-metrics text, after setting the pixels up to 320*256 under "Explore" screen.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66578]
## User Experience: 
Low-vision and users with cognitive disability will not be able to navigate through all the controls properly and will get challenges while navigating on the controls present on the page are overlapped with each other and are getting misplaced after setting the pixels at 320*256.

## Pre-requisite:  
Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

## Repro Steps:
Open #Url: https://play.grafana.org/. Sign in with valid credentials. "Home" page appears.
Navigate to the left navigation bar and to the "Explore" button, select it. New "Explore" page appears, Navigate it. Navigate to the left navigation bar and to the "Explore" button, select it. New "Explore" page appears, Navigate it. Navigate to the "Query pane's first row and expand it. Navigate to the "Query Patterns" combo box. Verify and observe the issue.

## Actual Result: 
Issue: Controls overlap with Grafana cloud-ml-metrics text, after setting the pixels up to 320*256 under "Explore" screen.
## Expected Result: 
Solution: Controls should not get overlapped with Grafana cloud-ml-metrics text and should be properly visible, after setting the pixels up to 320*256 under "Explore" screen.

**Comment by ifrost** [comment]
> Navigate to the "Query Patterns" combo box. Verify and observe the issue.

We don't have "Query Patterns" on Explore page, could this be part of a query editor? If yes, which one was used in the test?

**Comment by Elfo404** [comment]
closing this as i'm not able to reproduce. it's not clear what the issue is about (query rows control buttons? kickstart your query modal? prom query builder?)
